news/2020-02-03-Micro_Profile_OpenTracing_Comes_To_WildFly.adoc

```diff
@@ -0,0 +1,123 @@
+= Micro Profile OpenTracing comes to WildFly =
+ehsavoie
+2020-02-17
+:revdate: 2020-02-03 08:00:00 -0600
+:awestruct-tags: [wildfly, microprofile, configuration, opentracing]
+:awestruct-layout: blog
+:source-highlighter: coderay
+:encoding: utf-8
+:lang: en
+:awestruct-description: How to configure and use Micro Profile OpenTracing within WildFly
+
+Until WildFly 19 you could use MPOT (Micro Profile OpenTracing) to trace your application using environment variables. With wildFly 19 you can now configure several Jaeger Tracers to be used in your applications.
+
+== Installing Jaeger
+
+Let's start a jaeger instance using docker :
+[source,bash]
+--
+docker run -d --name jaeger \
+  -p 16831:6831/udp \
+  -p 5778:5778 \
+  -p 14268:14268 \
+  -p 16686:16686 \
+  jaegertracing/all-in-one:1.16
+--
+
+Now, you can navigate to link:http://localhost:16686[http://localhost:16686] to access the Jaeger UI.
+
+[NOTE]
+====
+For demonstration purpose we have mapped the port *6831* to *16831*.
```

Comment:
s/purpose/purposes

Perhaps add a sentence explaining why a little bit?  I read 'http://localhost:16686' and then there is this not talking about 16831 which is unrelated to the sentence above. So it's confusing. A sentence about why would both help clarify that the Note isn't directly about the previous line, plus it just helps clarify why.

Reply:
I'll just remove this as it doesn't demonstrate anything interesting